draft-normington-jsonpath.md

```diff
@@ -403,7 +411,7 @@ in {{RFC3629}}.
 A JSON value is logically a tree of nodes.
 
 Each node holds a JSON value (as defined by {{RFC8259}}) of one of the
-types object, array, number, string, or one of the literals `true`,
+types JSON object, array, number, string, or one of the literals `true`,
```

Comment:
Why are we specifying JSON for objects everywhere?  What other meaning does "object" have that warrants this specificity?

Reply:
I agree. Also other primitives like number and string are also _JSON_ numbers and _JSON_ strings, (with all the subtleties that come from that, both from spec and underlying json engine implementation choices allowed by the JSON spec)

Reply:
I changed them all to object.  I continue to believe this is bad usage, but if the spec is mainly directed at people who work exclusively with JSON, that may be a simplification.

Reply:
My feeling is that this is a JSON-related specification, so it's likely to be assumed.  We _could_ add a statement in the declarations at the beginning to the effect of, "All usages of 'object', etc, mean the JSON usage unless otherwise specified."